src/Entity/Cliente.php

```diff
@@ -15,7 +16,7 @@ class Cliente implements \JsonSerializable
     #[ORM\Column]
     private int $id;
 
-    #[ORM\Column]
+    #[ORM\Column(type: "boolean", options: ["default" => true])]
```

Comment:
ele pega da propriedade
```suggestion
    #[ORM\Column(options: ["default" => true])]
```